src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -1243,6 +1265,12 @@ public override void VisitDiscardOperation(IDiscardOperation operation)
             Assert.Equal(operation.Type, discardSymbol.Type);
         }
 
+        public override void VisitDiscardPattern(IDiscardPatternOperation operation)
```

Comment:
>VisitDiscardPattern [](start = 29, length = 19)

Can we assert something about DeclaredSymbol and MatchedType?

Reply:
No, as there are no such properties.

---
In reply to: [244207839](https://github.com/dotnet/roslyn/pull/32018#discussion_r244207839) [](ancestors = 244207839)